tests/functional/aws-node-sdk/test/bucket/deleteCors.js

```diff
@@ -0,0 +1,87 @@
+import assert from 'assert';
+
+import withV4 from '../support/withV4';
+import BucketUtility from '../../lib/utility/bucket-util';
+
+const bucketName = 'testdeletecorsbucket';
+const sampleCors = { CORSRules: [
+  { AllowedMethods: ['PUT', 'POST', 'DELETE'],
+    AllowedOrigins: ['http://www.example.com'],
+    AllowedHeaders: ['*'],
+    MaxAgeSeconds: 3000,
+    ExposeHeaders: ['x-amz-server-side-encryption'] },
+  { AllowedMethods: ['GET'],
+    AllowedOrigins: ['*'],
+    AllowedHeaders: ['*'],
+    MaxAgeSeconds: 3000 },
+] };
+
+const itSkipIfAWS = process.env.AWS_ON_AIR ? it.skip : it;
+
+describe('DELETE bucket cors', () => {
+    withV4(sigCfg => {
+        const bucketUtil = new BucketUtility('default', sigCfg);
+        const s3 = bucketUtil.s3;
+        const otherAccountBucketUtility = new BucketUtility('lisa', {});
+        const otherAccountS3 = otherAccountBucketUtility.s3;
+
+        describe('without existing bucket', () => {
+            it('should return NoSuchBucket', done => {
+                s3.deleteBucketCors({ Bucket: bucketName }, err => {
+                    assert(err);
+                    assert.strictEqual(err.code, 'NoSuchBucket');
+                    assert.strictEqual(err.statusCode, 404);
+                    return done();
+                });
+            });
+        });
+
+        describe('with existing bucket', () => {
+            beforeEach(() => s3.createBucketAsync({ Bucket: bucketName }));
+            afterEach(() => bucketUtil.deleteOne(bucketName));
+
+            describe('without existing cors configuration', () => {
+                it('should return a 200 response', done => {
+                    s3.deleteBucketCors({ Bucket: bucketName }, err => {
+                        assert.strictEqual(err, null,
+                            `Found unexpected err ${err}`);
+                        return done();
+                    });
+                });
+            });
+
+            describe('with existing cors configuration', () => {
+                beforeEach(done => {
+                    s3.putBucketCors({ Bucket: bucketName,
+                    CORSConfiguration: sampleCors }, done);
+                });
+
+                it('should delete bucket configuration successfully', done => {
+                    s3.deleteBucketCors({ Bucket: bucketName }, err => {
+                        assert.strictEqual(err, null,
+                            `Found unexpected err ${err}`);
+                        s3.getBucketCors({ Bucket: bucketName }, err => {
+                            assert.strictEqual(err.code,
+                                'NoSuchCORSConfiguration');
+                            assert.strictEqual(err.statusCode, 404);
+                            return done();
+                        });
+                    });
+                });
+
+                // Skip if AWS because AWS Node SDK raises CredentialsError
+                // before letting the request hit the API
```

Comment:
I assumed it was a sdk error because CredentialsError is unfamiliar and we usually return AccessDenied, but thanks for probing so I would confirm in more detail. This is the error returned:

`{ [SharedIniFileCredentialsProviderFailure: Missing credentials in config]
  message: 'Missing credentials in config',
  code: 'CredentialsError',
  time: Tue Jan 10 2017 15:43:12 GMT-0800 (PST),
  originalError: 
   { message: 'Could not load credentials from SharedIniFileCredentials',
     code: 'CredentialsError',
     time: Tue Jan 10 2017 15:43:12 GMT-0800 (PST),
     originalError: 
      { message: 'Profile lisa not found in /Users/electra/.aws/scality',
        code: 'SharedIniFileCredentialsProviderFailure',
        time: Tue Jan 10 2017 15:43:12 GMT-0800 (PST) } } }`

The node sdk relies on a config file to pull authentication data; when testing S3 Server we've configured to pull this data from conf/authdata.json in the s3 repo. When testing with AWS_ON_AIR it pulls from ~/.aws/scality, which should contain matching profiles with real AWS credentials for the tests to pass. 

In order for this test to always pass with AWS_ON_AIR we would have to assume everyone who runs the tests has updated their local ~/aws/scality file to contain a second set of credentials in addition to the default under a 'lisa' profile. I will add a comment to explain this in case someone does want to run the test but think it's safer to keep `itSkipIfAWS` setting by default.